java/mapreduce/src/java/org/apache/orc/mapred/OrcInputFormat.java

```diff
@@ -58,7 +58,7 @@
    */
   public static boolean[] parseInclude(TypeDescription schema,
                                        String columnsStr) {
-    if (columnsStr == null ||
+    if (StringUtils.isBlank(columnsStr) ||
```

Comment:
Hi @dongjoon-hyun,

Thank you for reviewing the patch and pointing me in the right direction. I had earlier fixed only the exception. Based on your comment, a bit of reading through the code and the spark test that you have linked in the JIRA my understanding is that the desired behavior is to not select/project any columns when the `orc.include.columns` is a blank string. I have added a test case for what I understand is the desired behavior. 

I have left the behavior for null(default) value to be as before which seems bit inconsistent(null & blank should not be different) but I am not sure on why it was changed and if it's ok to modify that behavior.

I am completely new to ORC and this is my first patch so my understanding might be completely off. Please feel free to point me in the right direction. Thanks!